app/components/DividerComponent.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
```

Comment:
Can we change the filename of this? I don't think we need `Component` at the end, it reads `app/components/DividerComponent.js`. ie the directory lets us know its a component.

Reply:
yes yes we can. will do asap and update